elastic4s-core/src/main/scala/com/sksamuel/elastic4s/requests/searches/aggs/TopHitsAggregation.scala

```diff
@@ -16,6 +16,7 @@ case class TopHitsAggregation(name: String,
                               version: Option[Boolean] = None,
                               scripts: Map[String, Script] = Map.empty,
                               storedFields: Seq[String] = Nil,
+                              docValueFields: Seq[String] = Nil,
                               subaggs: Seq[AbstractAggregation] = Nil,
                               metadata: Map[String, AnyRef] = Map.empty,
                               highlight: Option[Highlight] = None)
```

Comment:
can you move the new field to the end for slightly better backwards compatibility?

Reply:
Done